packages/insured-bridge-relayer/src/RelayerConfig.ts

```diff
@@ -13,6 +13,12 @@ export const bridgeDepositBoxDeployData = {
   288: { blockNumber: 223808 },
 };
 
+// Following settings can be overridden to optimize L1 client event search for select events. For example,
+// the "DepositRelayed" event search request could return > 10,000 return values so we need to shorten the block
+// search using these parameters because some node providers like Infura limit event search return values to 10,000.
+export const bridgePoolEarliestBlockToSearch = 13545377;
```

Comment:
How did you determine this number?

Reply:
oldest spoke pool's deployment block number, so the weth pool's deploy block. No events for any spoke pools should be before this

Reply:
in v2 we have this on a per spoke basis which helps, rather than grouping them all together.

Reply:
I think its actually pretty safe to group them all together since the BridgePools are always deployed to L1 and we can guarantee no Relay events before this block, and the lookback is ~1.1 million blocks to this block which should be fine for quite some time